### DISCUSSION TITLE
Hello, I am Kalhara. Follow me on GitHub

### DISCUSSION THREAD

**Discussion by KalharaBatangala** [community/community#138621] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Show & Tell

### Body
Hello guys, I'm an Engineering Undergraduate at University of Sri Jayewardenepura, Sri Lanka. I am researching on developing Deep neural networks, image processing and machine learning algorithms. Follow me on GitHub who are AI enthusiasts and love to work on ML and DL projects together. 

👉https://github.com/KalharaBatangala?tab=overview&from=2024-09-01&to=2024-09-14

**Comment by ebndev** [comment, marked as answer]
Welcome to GitHub and the GitHub Community @KalharaBatangala 🎆! We're glad you're here and can't wait to see where your GitHub journey takes you.

To help keep our space clutter-free, we recommend introducing yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204) and connect with other users there! 👋
